### DISCUSSION TITLE
Not showing in integrations tab

### DISCUSSION THREAD

**Discussion by hancooo** [petretiandrea/home-assistant-tapo-p100#241] [Q&A]
Hi pal. Followed all the details and installed successfully, but not showing in the integrations tab. Any ideas? 
Thanks in advanced. 
(running fresh install of Home Assistant on latest version on Pi 4)

**Comment by LudiusB** [comment]
same problem here, running home Assistant OS 8.5 (2022.8.2) in Proxmox VM.

**Comment by LudiusB** [comment]
GOT IT: in google chrome open the developer mode after that klicking 2-3 seconds on de reload button there is a "reaload an clear cache" option. after done this i got my Tapo p110 controler integration to add.

hope this helps
